packages/components/src/templates/next/components/internal/UnsupportedBrowserBanner/UnsupportedBrowserBanner.tsx

```diff
@@ -19,13 +20,32 @@ const getUserAgent = (): string | undefined => {
 const isSupportedBrowser = ({
   userAgent,
 }: UnsupportedBrowserBannerProps): boolean => {
+  if (userAgent === undefined) return true
+  if (userAgent === "") return true
```

Comment:
why would both of these be `true`? i'm assumign the reason here is that these are bots and we just ignore? additionally, should we also ignore `null`?

Reply:
anyway, don;t think we need `null` but added in just in case haha